src/Package/Orgs/Members.php

```diff
@@ -232,4 +232,123 @@ public function conceal($org, $user)
 			204
 		);
 	}
+
+	/**
+	 * Get organization membership
+	 *
+	 * In order to get a user's membership with an organization, the authenticated user must be an organization owner.
+	 *
+	 * @param   string  $org   The name of the organization.
+	 * @param   string  $user  The name of the user.
+	 *
+	 * @return  object
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getMembership($org, $user)
+	{
+		// Build the request path.
+		$path = '/orgs/' . $org . '/memberships/' . $user;
+
+		return $this->processResponse($this->client->get($this->fetchUrl($path)));
+	}
+
+	/**
+	 * Add or update organization membership
+	 *
+	 * In order to create or update a user's membership with an organization, the authenticated user must be an organization owner.
+	 *
+	 * @param   string  $org   The name of the organization.
+	 * @param   string  $user  The name of the user.
+	 *
+	 * @return  object
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function updateMembership($org, $user)
+	{
+		// Build the request path.
+		$path = '/orgs/' . $org . '/memberships/' . $user;
+
+		return $this->processResponse($this->client->put($this->fetchUrl($path)));
+	}
```

Comment:
I believe this method is missing some data...
https://developer.github.com/v3/orgs/members/#add-or-update-organization-membership